{{cookiecutter.project_slug}}/.travis.yml

```diff
@@ -1,34 +1,56 @@
 # Config file for automatic testing at travis-ci.org
 # This file will be regenerated if you run travis_pypi_setup.py
 
-language: python
-python: 3.5
+sudo: false
+
+language: generic
 
 env:
-  - TOXENV=py35
-  - TOXENV=py34
-  - TOXENV=py33
-  - TOXENV=py27
-  - TOXENV=py26
-  - TOXENV=pypy
+  - PYVER="3.5"
+  - PYVER="3.4"
+  - PYVER="2.7"
+
+install:
+  # Install Miniconda.
+  - curl -L https://repo.continuum.io/miniconda/Miniconda3-4.1.11-Linux-x86_64.sh > miniconda.sh
+  - openssl md5 miniconda.sh | grep 874dbb0d3c7ec665adf7231bbb575ab2
+  - bash miniconda.sh -b -p ~/miniconda
+  - rm miniconda.sh
+  - source ~/miniconda/bin/activate root
+  - conda config --set show_channel_urls true
+  - conda update -y -n root --all
+  - conda install -y -n root conda-build
+  - conda clean -tipsy
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -U tox
+  # Create the test environment.
+  - conda create -y -n testenv python="${PYVER}"
+  - conda remove -y --force -n testenv openssl
```

Comment:
Have to remove `defaults` copy of `openssl` or we run into issue ( https://github.com/conda/conda/issues/3319 ).